src/reducers/contacts.reducer.spec.js

```diff
@@ -0,0 +1,147 @@
+import {
+  increaseLikes,
+  decreaseLikes,
+  deleteContact,
+  deleteSelectedContacts,
+  clearContacts,
+  changeIsChecked,
+  fetchFavourites,
+  fetchContacts,
+  addContact
+} from '../actions';
+import contactsReducer from './contacts.reducer';
+import randomContactsList from '../data/data';
+import { getRandomNumber, getFavouritesList } from '../helpers/helper';
+import { DEFAULT_NUMBER_OF_CONTACTS as N } from '../constants/constants';
+
+describe('contacts reducer with id param', () => {
+
+  const state = {
+    contacts: randomContactsList,
+    favourites: []
+  };
+  let randomContactIndex;
+  let randomContact;
+
+  beforeEach(() => {
+    randomContactIndex = getRandomNumber(0, N);
+    randomContact = state.contacts[randomContactIndex];
+  });
+
+  test('increased likes', () => {
```

Comment:
the names for the tests should have the following format:
describe('(method you call)')
test('should (what you do in the expect) when (if you have something mocked specially for this method)').
in your case, a proper name for this method would be:
test('increaseLikes should increase number of likes').
if you would have multiple tests for the same method, you add the describe like this
describe('increaseLikes')
test('should ....')

Reply:
please use this naming conventions for all the unit tests